addons/mail/static/src/core/web/activity.js

```diff
@@ -115,4 +117,10 @@ export class Activity extends Component {
     get thread() {
         return this.threadService.getThread(this.props.data.res_model, this.props.data.res_id);
     }
+
+    get avatarImg() {
+        return imageUrl("res.users", this.activity.user_id[0], "avatar_128", {
+            unique: this.activity.user_id[2].write_date,
```

Comment:
it would  be better if we could save the user data into actual user/persona rather than keeping the strange format